src/constants.ts

```diff
@@ -0,0 +1,31 @@
+export interface ModelInfo {
```

Comment:
this should just be the props for ViewerTitle (which could also be renamed because it's handling all this stuff. Since it's only being used in that file, I would just put it in the ViewerTitle file